python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -413,28 +413,22 @@ class RandomForestClassifier(Base, ClassifierMixin):
         assert self.treelite_serialized_model or self.rf_forest, \
             "Attempting to create treelite from un-fit forest."
 
-        if self.num_classes > 2:
-            raise NotImplementedError("Pickling for multi-class "
-                                      "classification models is currently not "
-                                      "implemented. Please check cuml issue "
-                                      "#1679 for more information.")
         if self.treelite_serialized_model:  # bytes -> Treelite
             tl_handle = <ModelHandle><uintptr_t>treelite_deserialize(
                 self.treelite_serialized_model)
         else:                 # RF -> Treelite
-            task_category = CLASSIFICATION_MODEL
             build_treelite_forest(
                 &tl_handle,
                 rf_forest,
                 <int> self.n_cols,
-                <int> task_category)
+                <int> self.num_classes)
         self.treelite_handle = <uintptr_t>tl_handle
         return self.treelite_handle
 
     def _get_serialized_model(self):
         """
         Returns the self.treelite_serialized_model.
-        Cuml RF model gets converted to treelite bytes by:
+        Cuml RF model gets converted to treelite protobuf bytes by:
```

Comment:
Can we remove the mention of protobuf? Protobuf is no longer required component of cuML.

Reply:
Also see #2538, which removed Protobuf from cuML.